airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -2093,7 +2089,7 @@ def create_empty_dataset(self, *args, **kwargs) -> None:
             DeprecationWarning, stacklevel=3)
         return self.hook.create_empty_dataset(*args, **kwargs)
 
-    def get_dataset_tables(self, *args, **kwargs) -> Dict[str, Union[str, int, List]]:
+    def get_dataset_tables(self, *args, **kwargs) -> List[Dict[str, Any]]:
```

Comment:
Should change of returned type in these deprecated methods (`get_dataset_tables` and `get_dataset_tables_list`) be reflected in UPDATING.md as well?

Reply:
This method is deprecated, and from what I see this is not change but a fix (List[Dict] is mentioned in the hook method)

Reply:
But shouldn't returned types be `List[TableListItem]` for `get_dataset_tables` and `List[Dict[str, Any]]` for `get_dataset_tables_list` then?

Reply:
Fixed